src/frontend/src/tests/pages/ProjectDetailPage/TaskList.test.tsx

```diff
@@ -31,10 +32,22 @@ const mockAuthHook = (user = exampleAdminUser) => {
 };
 
 jest.mock('../../../hooks/tasks.hooks');
+const mockedCreateTask = useCreateTask as jest.Mock<UseMutationResult>;
+const mockCreateTaskHook = (isLoading: boolean, isError: boolean, error?: Error) => {
```

Comment:
so look at mock-hooks.ts in the test-support folder and then see how i use stuff like mockEditProjectReturnValue in tests

imo it's easier because it cuts way down on # of lines